test/test_cursor.py

```diff
@@ -1222,6 +1223,64 @@ def assertCursorKilled():
         else:
             self.assertEqual(0, len(listener.started_events))
 
+    @client_context.require_failCommand_appName
+    def test_timeout_kills_cursor_asynchronously(self):
+        # Kill any cursors possibly queued up by previous tests.
+        gc.collect()
+        self.client._process_periodic_tasks()
+
+        listener = AllowListEventListener("killCursors")
+        client = rs_or_single_client(event_listeners=[listener])
+        self.addCleanup(client.close)
+        coll = client[self.db.name].test_timeout_kills_cursor
+
+        # Add some test data.
+        docs_inserted = 2
+        coll.insert_many([{"i": i} for i in range(docs_inserted)])
+
+        listener.reset()
+
+        cursor = coll.find({}, batch_size=1)
+        cursor.next()
+
+        # Ensure the cursor times out.
+        delay_get_mores = {
```

Comment:
Could you rename this var and comment? I was initially confused because we aren't actually delaying anything, instead we're mocking a MaxTimeMSExpired error.